_ml-commons-plugin/model-serving-framework.md

```diff
@@ -110,18 +120,22 @@ OpenSearch responds with the `model_id`:
 
 ### Load model from index
 
-With the `model_id`, you can now load the model from the model's index to deploy the model with ML nodes. The load API operation reads model chunks from the model index, then create an instance of that model and save the chunks in the ML node's cache.
+With the `model_id`, you can now load the model from the model's index to deploy the model to ML nodes. The load API reads model chunks from the model index, then creates an instance of that model and save the model instance in the ML node's cache.
+
+
+Add the `model_id` to the load API. 
 
-Add the `model_id` to the load API operation. 
 
 ```json
 POST /_plugins/_ml/models/<model_id>/_load
 ```
 
 By default the ML Common's setting `plugins.ml_commons.only_run_on_ml_node` is set to `false`. When `false`, models load on ML nodes first. If no ML nodes exist, models load on data nodes. When running ML models in production, set `plugins.ml_commons.only_run_on_ml_node` to `true`, so that models only load on ML nodes.
 
+
 #### Sample Request: Load into any available ML node
 
+
 In this example request, OpenSearch loads the model into any available OpenSearch node. 
```

Comment:
"on" instead of "into"? This should end with a colon if it introduces a request.

Reply:
Into might make more sense here from a technical perspective, because when the model loads into a node, we're loading the model into memory.